src/App.jsx

```diff
@@ -2,7 +2,7 @@ import './App.css';
 import SocialMediaBar from './components/SocialMediaBar';
 import Carousel from './components/Carousel';
 import Navbar from './components/NavbarComponent';
-import RecipeCard from './components/RecipeCard';
+//import RecipeCard from './components/RecipeCard';
```

Comment:
You might have forgotten to delete this import

Reply:
thank you for pointing that out!